### PR TITLE
Replaces the `h2` tags with the `PageHeading` component.

### LINKED ISSUE #732: Create Heading component
**Current behavior:**

Every component looks different.

**Expected behavior:**

Should look similar, at least h2 with some font style

Image

Image

Image

etc.

We need always h2 headers that are formatted similar.

### PR DESCRIPTION
**Pull request by splendidcomputer**
fixes #732

### CODE REVIEW

File: apps/web/frontend/components/pageHeading.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+interface Props {
```
Comment (moncapitaine) [inline apps/web/frontend/components/pageHeading.tsx:3]: name the interface like the others do.

File: apps/web/frontend/components/projectForm/projectForm.tsx [hunk 2]
```diff
@@ -104,7 +104,7 @@ export const ProjectForm = (props: ProjectFormProps): JSX.Element => {
       onSubmit={handleSubmit(handleSubmitHelper)}
     >
       {isNewProject ? (
-        <h2 className="w-full text-lg font-semibold text-gray-400 dark:text-white">Create new project</h2>
+        <h2 className="w-full text-lg font-semibold text-black dark:text-white">Create new project</h2>
```
Comment (moncapitaine) [inline apps/web/frontend/components/projectForm/projectForm.tsx:107]: use the pageHeading?

File: apps/web/frontend/components/timeSheetView/workHoursSheet.tsx [hunk 3]
```diff
@@ -31,10 +32,10 @@ export const WorkHoursSheet = (): JSX.Element => {
     <div className="flex flex-col">
       <section className="mt-10 grid w-full grid-cols-3 gap-2 text-left">
         <article className="contents border-y text-lg">
-          <strong>Project</strong>
-          <strong>Task</strong>
-          <strong>Person</strong>
-          <strong>Hours</strong>
+          <PageHeading>Project</PageHeading>
```
Comment (moncapitaine) [inline apps/web/frontend/components/timeSheetView/workHoursSheet.tsx:35]: this is not PageHeading.

add PageHeading to the pages/sheet.page.tsx and not here.

File: apps/web/frontend/components/projectForm/projectForm.tsx (hunk 2 above)
Comment (splendidcomputer) [reply apps/web/frontend/components/projectForm/projectForm.tsx:107]: Done

File: apps/web/frontend/components/reportForm/reportForm.tsx [hunk 4]
```diff
@@ -103,9 +104,9 @@ export const ReportForm = () => {
     <>
       <div>
         <div className="flex items-center justify-between">
-          <h1 className="my-4 text-2xl font-bold">
+          <PageHeading>
```
Comment (moncapitaine) [inline apps/web/frontend/components/reportForm/reportForm.tsx:107]: can we move outside the div?

File: apps/web/pages/projects/index.page.tsx [hunk 5]
```diff
@@ -60,7 +61,7 @@ const Projects = (): JSX.Element => {
     <ProtectedPage>
       <article>
         <div className="flex justify-between">
-          <h2 className="text-lg font-semibold text-gray-400 dark:text-white">Projects</h2>
+          <PageHeading>Projects</PageHeading>
```
Comment (moncapitaine) [inline apps/web/pages/projects/index.page.tsx:64]: can we move outside?

File: apps/web/frontend/components/reportForm/reportForm.tsx (hunk 4 above)
Comment (splendidcomputer) [reply apps/web/frontend/components/reportForm/reportForm.tsx:107]: It seems no.

File: apps/web/frontend/components/timeSheetView/workHoursSheet.tsx [hunk 6]
```diff
@@ -4,6 +4,7 @@ import useInfiniteScroll from 'react-infinite-scroll-hook'
 
 import { Spinner } from '@progwise/timebook-ui'
 
+import { PageHeading } from '../pageHeading'
```
Comment (PascalHelbig) [inline apps/web/frontend/components/timeSheetView/workHoursSheet.tsx:7]: I guess this import is no longer needed? 🙂

File: apps/web/pages/week/index.page.tsx [hunk 7]
```diff
@@ -39,7 +40,7 @@ const WeekPage = (props: WeekPageProps) => {
 
   return (
     <ProtectedPage>
-      <h2>Week entries</h2>
+      <PageHeading>Week enteries</PageHeading>
```
Comment (PascalHelbig) [inline apps/web/pages/week/index.page.tsx:43]:
```suggestion
      <PageHeading>Week entries</PageHeading>
```

**Review by PascalHelbig — CHANGES_REQUESTED** [review]
Hey @splendidcomputer, looks very good already. I checked out your branch and I noticed that there are still 7 `h2` tags. Of course, they need to be migrated as well.

image

File: apps/web/pages/week/index.page.tsx (hunk 7 above)
Comment (splendidcomputer) [reply apps/web/pages/week/index.page.tsx:43]: You are right.

File: apps/web/frontend/components/timeSheetView/workHoursSheet.tsx (hunk 6 above)
Comment (splendidcomputer) [reply apps/web/frontend/components/timeSheetView/workHoursSheet.tsx:7]: You are right. I removed it.

**Comment by splendidcomputer** [comment]
Replaces 6 `h2` tags with the `PageHeading`.

**Review by PascalHelbig — APPROVED** [review]
Good job, thanks!

**Review by moncapitaine — APPROVED** [review]

**Pull request merged**